smrf/tests/envphys/solar/test_toporad.py

```diff
@@ -106,16 +100,13 @@ def test_toporad(self):
             self.cosz,
             surface_albedo=0.5)
 
-        self.assertAlmostEqual(706.0067021452699, np.mean(trad_beam), places=4)
-        self.assertAlmostEqual(85.17264068026927, np.min(trad_beam), places=4)
-        self.assertAlmostEqual(1164.002943767303, np.max(trad_beam), places=4)
+        np.testing.assert_allclose(706, np.mean(trad_beam), atol=1)
+        np.testing.assert_allclose(85, np.min(trad_beam), atol=1)
+        np.testing.assert_allclose(1162, np.max(trad_beam), atol=1)
```

Comment:
To protocol this a bit. I had to bump up a few values by 1 and this is the only case with 2.

Reply:
Based on the figures above, I'm not surprised as there should be some differences. We'll fully address this with #204